java/arcs/android/storage/service/BindingContext.kt

```diff
@@ -68,6 +68,7 @@ class BindingContext(
     parentCoroutineContext: CoroutineContext,
     /** Sink to use for recording statistics about accessing data. */
     private val bindingContextStatisticsSink: BindingContextStatisticsSink,
+    private val devToolsProxy: IDevToolsProxy.Stub?,
```

Comment:
Should probably be just `IDevToolsProxy` or `DevToolsProxyImpl`, rather than the stub.